R/safe2x2Test.R

```diff
@@ -1550,19 +1556,50 @@ simulateWorstCaseQuantileTwoProportions <- function(na, nb, priorValues,
     currentPower <- mean(stopEs >= 1/alpha)
 
     #we look for the worst case (1-beta)% stopping time or power: store only that one
+    #also store the standard deviation, obtained through bootstrapping
+    #note that we do this only if we have found a new worst case: omit bootstrapping for every
+    #distribution.
     #also store the implied target belonging to the worst case power: exp(Expected [log E-waarde])
     if (currentQuantile >= currentWorstCaseQuantile) {
       currentWorstCaseQuantile <- currentQuantile
+      if (expectedStopTime) {
+        bootResult <- boot::boot(
+          data = stoppingTimes,
+          statistic = function(x, idx) {
+            mean(x[idx])
+          },
+          R = bootN
+        )
+      } else {
```

Comment:
I'll write a generic wrapper for this. Add a todo here so it's easier to swap. I'm not sure why the bootstrap is run while you run through the loop though.

Reply:
we finally report the nPlan for the worst case distribution in H1(delta), so need to store that one and their SE. Could store the stopping times and E variables for worst distribution until end of loop. Have added todo, let's do this together with the beautifying in the next iteration of the code. If we do not make simulation number M too high this won't take up too much memory.